runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/decompResolveFrame/decomp003.c

```diff
@@ -22,7 +22,9 @@
 
 #include "jvmti_test.h"
 #include <string.h>
+#if defined(J9ZOS390)
 #include <strings.h>
+#endif
```

Comment:
Please repeat the condition here:
```c
#endif /* defined(J9ZOS390) */
```
(in both files)

Reply:
Done